Atomic/util.py

```diff
@@ -472,3 +486,11 @@ def validate_manifest(spec, img_rootfs=None, img_tar=None, keywords=""):
     if keywords:
         cmd += ['-k',keywords]
     return subp(cmd)
+
+def get_signature_path(atomic_conf):
+    # If signature path is defined, get it; else return default
+    signature_path = get_atomic_config_item(['default-sigstore-path'], atomic_config=atomic_conf)
+    if signature_path is None:
+        signature_path = '/etc/pki/containers/'
+    return signature_path
```

Comment:
… and the per-registry/per-namespace/per-repository lookup from `lookaside.go` is necessary as well; we need various ISVs who publish to Docker hub to be able to maintain their own sigstore servers.